### PR TITLE
Fix redirect reference

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

Fixes redirect link for launch checklist.

## Affected DevDocs pages

- https://devdocs.magento.com/cloud/live/site-launch-checklist.html

### CODE REVIEW

**Comment by dshevtsov** [comment]
> Redirects are performed by serving an HTML file with an HTTP-REFRESH meta tag which points to your destination. No .htaccess file, nginx conf, xml file, or anything else will be generated. It simply creates HTML files.
https://github.com/jekyll/jekyll-redirect-from#how-it-works

The redirected file is not HTML, and thus it cannot be simply replaced by HTML file with HTTP-REFRESH meta tag.
Such kind of redirect can be performed by a web server means only. Feel free to create an internal internal ticket for this.

**Comment by dshevtsov** [comment]
Actually, this PR can be used to remove the existing redirect to .pdf if you don't mind, @meker12

File: src/cloud/live/site-launch-checklist.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ functional_areas:
   - Testing
   
 redirect_from:
-  - /common/pdf/magento-commerce-cloud-prelaunch-checklist.pdf
+  - https://devdocs.magento.com/common/pdf/magento-commerce-cloud-prelaunch-checklist.pdf
```
Comment (dshevtsov) [inline src/cloud/live/site-launch-checklist.md:9]:
```suggestion
```

**Comment by meker12** [comment]
Created a ticket to add the redirect to the server: https://jira.corp.magento.com/browse/MCLOUD-6054

**Comment by dshevtsov** [comment]
@meker12 I see that PR is closed, not merged. Why have you decided to not merge the change I suggested in this PR?

**Comment by meker12** [comment]
I will clean it up as part of the Cloud release.

**Pull request closed without merging**